packages/opencensus-web-instrumentation-zone-peer-dep/src/interaction-tracker.ts

```diff
@@ -35,7 +36,13 @@ declare const Zone: ZoneType & { prototype: Zone };
 // Delay of 50 ms to reset currentEventTracingZone.
 export const RESET_TRACING_ZONE_DELAY = 50;
 
+export interface TrackerOptions {
+  propagation?: Propagation;
+}
+
 export class InteractionTracker {
+  //
```

Comment:
Could you either remove this or fill out the comment?

Reply:
Removed.